test/conftest.py

```diff
@@ -0,0 +1,7 @@
+from backends import BACKENDS
```

Comment:
Do we need a file for this? Also, what does the file name represent?

Reply:
Oh so is this automatically generated? I was unfamiliar with this. If it is automatically generated, I don't see a need to carry it around.  But maybe I'm missing something?

Reply:
It is not autogenerated, I wrote it. It's required to accept CLI arguments.

Reply:
Gotcha. I can take a look at a later time to see if we can put it in the test script itself or change the name of the file.